sdks/python/apache_beam/typehints/decorators.py

```diff
@@ -340,6 +340,22 @@ def __repr__(self):
     return 'IOTypeHints[inputs=%s, outputs=%s]' % (
         self.input_types, self.output_types)
 
+  def __eq__(self, other):
```

Comment:
Why were these 3 methods needed?

Reply:
`__eq__` was needed for the test. Link complains if you don't then also supply `__ne__` and `__hash__`.